### DISCUSSION TITLE
v4版本的依赖问题

### DISCUSSION THREAD

**Discussion by srt180** [v2fly/v2ray-core#2984] [Q&A]
我现在有使用v2ray-core作为golang依赖的需求，鉴于v5版本的配置格式依旧在建设中，所以考虑v4版本的v2ray-core作为依赖。但是我发现v4版本的v2ray-core很久不更新了，其中还留存着go-quic的编译问题。我是否可以提个pr更新一下v4版本的依赖呢？

**Comment by Losalado** [comment]
v5版本也支持v4的格式吧，或许直接升级也可以
